### PR TITLE
Update README.md with link to HTML, PDF, CONTRIB

### PR DESCRIPTION
**Pull request by dsblank**
Also made it read a bit better in plain text mode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 
 [![Build Status](https://travis-ci.com/jupyter4edu/jupyter-edu-book.svg?branch=master)](https://travis-ci.com/jupyter4edu/jupyter-edu-book)
 
-An [open book about Jupyter and its use in teaching and learning](https://jupyter4edu.github.io/jupyter-edu-book/)
+An open book about Jupyter and its use in teaching and learning.
+
+This repository is the source materials for the book "Teaching and Learning with Jupyter." It can be found here: 
+https://jupyter4edu.github.io/jupyter-edu-book/
+
+To view the book as HTML, and download it is PDF and other formats, please visit:
```
Comment (willingc) [inline README.md]: small typo "is" -> "as"

Comment (dsblank) [reply README.md]: Good catch!

**Comment by dsblank** [comment]
Fix should be addressed. Thanks!

**Comment by willingc** [comment]
Thanks @dsblank

**Pull request merged**